cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -835,6 +835,14 @@
     </description>
   </property>
 
+  <property>
+    <name>log.kafka.topic.prefix</name>
+    <value>logs.user-v2</value>
+    <description>
+      Kafka log topic to publish logs
+    </description>
+  </property>
+
```

Comment:
Also, why are these "KAFKA_TOPIC_PREFIX", when in the code, they are always assigned to a Kafka Topic? Should property just drop the ".prefix" portion? The descriptions would then be "Kafka topic to publish logs"...